### DISCUSSION TITLE
Price display issue

### DISCUSSION THREAD

**Discussion by TehnikaSara** [PrestaShop/PrestaShop#34503] [Q&A]
Hello everyone,

I have a very strange issue with my PrestaShop 1.7 version. Sometimes, the price is displayed to the customer without tax. Whenever I check the shop, the price is correct with tax. However, it occasionally happens that a customer sends a picture of a product, and the price is displayed without tax. Tax rules are set up with only one tax, and it's assigned to all users.

The website is [www.tehnikasara.ba](http://www.tehnikasara.ba/).

I am attaching two images to this message from the situation I had today.

![2](https://github.com/PrestaShop/PrestaShop/assets/150246382/e2d84b3d-955a-494f-9061-466d908ef080)
![1](https://github.com/PrestaShop/PrestaShop/assets/150246382/2e1edeb2-eefc-45dc-8b9f-92232d15bf1b)

**Comment by TehnikaSara** [comment]
Can someone help me ? 👯

**Comment by MatShir** [comment]
Have already you check your users groups config ? The user group can have different tax displays
![Capture d’écran 2023-11-13 à 16 23 41](https://github.com/PrestaShop/PrestaShop/assets/43613217/782ed8be-75bc-4eb8-a9bc-5b0a1ef170b9)
If your issue is urgent, I highly recommend you to check to[ this page](https://www.prestashop-project.org/support/) to find support

**Comment by TehnikaSara** [comment]
Everythig is set up correctly :

![image](https://github.com/PrestaShop/PrestaShop/assets/150246382/2d7a63bd-b378-441e-8d98-ea03f19c339e)
